src/vaultRegistry/vault/getVaultName.ts

```diff
@@ -0,0 +1,17 @@
+import { Address } from "viem";
+import { IpfsClient } from "@/lib/ipfsClient";
+
+const AddressToVaultName: { [key: Address]: string } = {
+  "0x3D04Aade5388962C9A4f83B636a3a8ED63ea5b4D": "USDT Junior Tranche",
+  "0x11E10B12e8DbF7aE44EE50873c09e5C7c3e01385": "USDT Senior Tranche",
+  "0x52Aef3ea0D3F93766D255A1bb0aA7F1C4885E622": "USDC Junior Tranche",
+  "0xcdc3CbF94114406a0b59aDA090807838369ced2b": "USDC Senior Tranche",
+  "0x6cE9c05E159F8C4910490D8e8F7a63e95E6CEcAF": "DAI Junior Tranche",
+  "0x30D6a7B8c985d5dd7B9823d3B6Ae2726c8FFf81F": "DAI Senior Tranche"
+}
```

Comment:
is this only supposed to be used for idle?

If not, we need to distinguish between the different chains.